src/Sofort/SofortLib/TransactionData.php

```diff
@@ -567,7 +567,21 @@ public function setStatus($status)
         
         return $this;
     }
-    
+
+
+    /**
+     * Request for transactions via projectId
+     *
+     * @param string $projectId
+     * @return TransactionData
+     */
+    public function setProjectId($projectId)
+    {
+        $this->_parameters['project_id'] = $projectId;
+        $this->_projectId = $projectId;
```

Comment:
what's this class prop for? I also didn't find any initialization of it. Might be redundant.

Reply:
nvm found it in abstractWrapper